code/game/objects/items/weapons/twohanded.dm

```diff
@@ -625,13 +625,21 @@
 		if(!isliving(target))
 			return
 		else
-			target.Weaken(8 SECONDS)
+			user.apply_status_effect(STATUS_EFFECT_CHAINSAW_SLAYING)
+			target.KnockDown(8 SECONDS)
 			..()
```

Comment:
one thing thats always bugged me, is that this `target.KnockDown()` bipasses all block chance/energy shields.
I think this can be altered if you make it check `else if(..())` as `..()` returns `. = M.attacked_by(src, user, def_zone)`
or 
you might want to make it so you always get the status effect regardless of blocking it.
then you can just do
```
else
    apply_effect()
    if(..())
        KnockDown()
```

Reply:
will do

Reply:
Done!